java/src/jmri/jmrix/openlcb/swing/monitor/MonitorPane.java

```diff
@@ -33,15 +34,27 @@ public class MonitorPane extends jmri.jmrix.AbstractMonPane implements CanListen
     public MonitorPane() {
         super();
         pm = InstanceManager.getDefault(UserPreferencesManager.class);
+        tagManager = InstanceManager.getDefault(IdTagManager.class);
     }
 
     CanSystemConnectionMemo memo;
     AliasMap aliasMap;
     MessageBuilder messageBuilder;
     OlcbInterface olcbInterface;
+
+    IdTagManager tagManager;
+    static final String tagPrefix = jmri.jmrix.openlcb.swing.eventtable.EventTablePane.tagPrefix;
```

Comment:
I think this constant should be in jmri.jmrix.openlcb.SomeClass. The tag prefix isn't limited to swing. For example, someone might want to use the web server and json for this.

Reply:
Moved to the jmrix/openlcb/Bundle.  Thanks for the suggestion!

Reply:
Sorry for complaining, but would it be an issue if someone either changes this string in the bundle, or translates it to another language? For example, if a user uses German language and have this string translated to German, and then requests support on the JMRI users list and an English speeker developer tries to load the panel file?

I would suggest putting this string in a class like [OlcbConstants.java](https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrix/openlcb/OlcbConstants.java) or [OlcbAddress.java](https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrix/openlcb/OlcbAddress.java).